### PR TITLE
lesson_9

### PR DESCRIPTION
**Pull request by alisa-bielenko**

### CODE REVIEW

File: lesson_9/HW9/index9HW.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      </table>
+
+      <div class="s">
+        <div class="s2">
+          <h2><u>Skills:</u></h2>
+        </div>
+        <div class="s1">
+          <h3>
+            <ul>
+              <b>
```
Comment (RomanIlkiv) [inline lesson_9/HW9/index9HW.html:45]: use css already instead of styles tag

File: lesson_9/HW9/index9HW.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        </tr>
+        <tr>
+          <td>2018-present</td>
+          <td class="position">Chief Accountant at <a href="https://www.google.com/">TEXNO-CENTER LTD</a></td>
+        </tr>
+      </table>
+
+      <div class="s">
+        <div class="s2">
+          <h2><u>Skills:</u></h2>
```
Comment (RomanIlkiv) [inline lesson_9/HW9/index9HW.html:40]: use css already instead of styles tag (I mean `<u>...</u>`)

File: lesson_9/HW9/index9HW.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          <td>2005-2018</td>
+          <td class="position">Chief Accountant at <a href="https://www.google.com/">IRSM LTD</a></td>
+        </tr>
+        <tr>
+          <td>2018-present</td>
+          <td class="position">Chief Accountant at <a href="https://www.google.com/">TEXNO-CENTER LTD</a></td>
+        </tr>
+      </table>
+
+      <div class="s">
```
Comment (RomanIlkiv) [inline lesson_9/HW9/index9HW.html:38]: rename to skills-container or something like that
classes should have some clear name to understand other people what this class is used for

File: lesson_9/HW9/index9HW.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <div class="s2">
```
Comment (RomanIlkiv) [inline lesson_9/HW9/index9HW.html:39]: skills-header

File: lesson_9/HW9/index9HW.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+          <h2><u>Skills:</u></h2>
+        </div>
+        <div class="s1">
```
Comment (RomanIlkiv) [inline lesson_9/HW9/index9HW.html:42]: skills-list

**Review by RomanIlkiv — CHANGES_REQUESTED** [review]

File: lesson_9/HW9/index9HW.html (hunk 2 above)
Comment (alisa-bielenko) [reply lesson_9/HW9/index9HW.html:40]: ok

File: lesson_9/HW9/index9HW.html (hunk 5 above)
Comment (alisa-bielenko) [reply lesson_9/HW9/index9HW.html:42]: ok

File: lesson_9/HW9/index9HW.html (hunk 4 above)
Comment (alisa-bielenko) [reply lesson_9/HW9/index9HW.html:39]: ok

File: lesson_9/HW9/index9HW.html (hunk 3 above)
Comment (alisa-bielenko) [reply lesson_9/HW9/index9HW.html:38]: ok

File: lesson_9/HW9/index9HW.html (hunk 1 above)
Comment (alisa-bielenko) [reply lesson_9/HW9/index9HW.html:45]: ok

**Pull request merged**